app/code/Magento/CmsGraphQl/etc/schema.graphqls

```diff
@@ -13,13 +13,16 @@ type StoreConfig @doc(description: "The type contains information about a store
 type Query {
     cmsPage (
         id: Int @doc(description: "Id of the CMS page")
+        identifier: String @doc(description: "Identifier of the CMS page")
     ): CmsPage @resolver(class: "Magento\\CmsGraphQl\\Model\\Resolver\\Page") @doc(description: "The CMS page query returns information about a CMS page")
     cmsBlocks (
         identifiers: [String] @doc(description: "Identifiers of the CMS blocks")
     ): CmsBlocks @resolver(class: "Magento\\CmsGraphQl\\Model\\Resolver\\Blocks") @doc(description: "The CMS block query returns information about CMS blocks")
 }
 
 type CmsPage @doc(description: "CMS page defines all CMS page information") {
+    page_id: Int @doc(description: "Entity ID of CMS page")
+    identifier: String @doc(description: "Identifier of the CMS page")
```

Comment:
Hi @atwixfirster - please check comment from @naydav about GraphQL scheme and apply needed changes.
Thx!
https://github.com/magento/graphql-ce/pull/585#issuecomment-484244309

Reply:
**implemented**

Please review!

Thank you!

Reply:
@vpodorozh , could you please pay your attention to this PR?

Thank you!